ts/Series/DataModifyComposition.ts

```diff
@@ -773,4 +785,22 @@ export default DataModifyComposition;
  * @apioption plotOptions.series.cumulative
  */
 
+/**
+ * Defines if cumulation should start from the first point within the visible
+ * range or should start from the first point **before** the range.
```

Comment:
```suggestion
 * range or should start from the last point **before** the range.
```

Reply:
I based my description on `compareStart` option. Should I change that description as well? The options are doing the same thing, so the description should be the same in both cases.

Reply:
Yes, I think we should change that too.